pkg/controller/services_test.go

```diff
@@ -185,33 +183,242 @@ func TestServiceIp(t *testing.T) {
 	cont.stop()
 }
 
+func TestContractScopeValues(t *testing.T) {
+	name := "testContract"
+	tenant := "testTenant"
+	graphName := "testGraph"
+
+	contract, err := apicContract(name, tenant, graphName, "global")
+	assert.Empty(t, err)
+	assert.Equal(t, contract.GetAttrStr("scope"), "global")
+
+	_, err = apicContract(name, tenant, graphName, "wrong-Name")
+	assert.NotNil(t, err)
+
+	contract, err = apicContract(name, tenant, graphName, "")
+	assert.Empty(t, err)
+	assert.Equal(t, contract.GetAttrStr("scope"), "context")
+
+	contract, err = apicContract(name, tenant, graphName, "GloBal")
+	assert.Empty(t, err)
+	assert.Equal(t, contract.GetAttrStr("scope"), "global")
```

Comment:
Is there an assert needed for the third legal value "tenant"?

Reply:
That's to make sure a blank scope value("") defaults to "context"